exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
 const name = " Daniel  ";
 
 console.log(message);
+
+//Unsure if how I can display the name length mid sentence, or do I need to create another string afterwards? Also trim what white space?
```

Comment:
Sinead you have to create another variable and use it to get the trimmed length of name.
`.trim()` actually remove all the extra spaces (as you see there are two tab spaces after Daniel and we only need one)
here's an example how to do it.


const myName = " Daniel       ";
const trimmed = myName.trim();
const greeting = "My name is ";
const middleText = " and my name is ";
const nameLength = trimmed.length;
const endText = " characters long";
const message = greeting + trimmed + middleText + nameLength + endText;
console.log(message);


there is another shorter way to consol this message.
let me know if you don't understand.
I'll explain with a screen share

Reply:
Thank you Haseeb :)